public/app/features/explore/Logs.tsx

```diff
@@ -251,6 +278,8 @@ export class Logs extends PureComponent<Props, State> {
           wrapLogMessage={wrapLogMessage}
           timeZone={timeZone}
           getFieldLinks={getFieldLinks}
+          orderLogs={sortLogRows}
```

Comment:
Do we need to pass this as prop? Is that just because one is in package and one in core? Personally I think we could move it to grafana-data package.